client/jetpack-cloud/sections/partner-portal/issue-license-v2/index.tsx

```diff
@@ -93,8 +93,8 @@ export default function IssueLicenseV2( { selectedSite, suggestedProduct }: Assi
 						<Subtitle>
 							{ translate( 'Select single product licenses or save when you issue in bulk' ) }
 						</Subtitle>
-						<Actions>
-							{ selectedLicenses.length > 0 && (
```

Comment:
It is not related to the main goal of this PR. This prevents the header from allocating a few spaces when there are no actions to display.